JS101/javascript-dev-tools.md

```diff
@@ -1,10 +1,24 @@
-this lesson is all about what you can do with chrome's devtools to debug and work with Javascript
+Knowing how to use the Developers Tools inside your browser is an important skill for any web developer. The Developer Tools are useful for running Javascript code, editing HTML and CSS Styles without having to refresh the page, or viewing performance data. They also save you lots of time. Getting started couldn't be any easier.
 
-shouldn't be tooo long... maybe jsut a couple of vids to watch.
+## Opening Dev Tools
 
-basic install list: https://github.com/jankeLearning/content-md/blob/master/tools/01-installations.md
-CLI intro: https://github.com/jankeLearning/content-md/blob/master/tools/01-cli.md
-a little bit on chromdev: https://github.com/jankeLearning/content-md/blob/master/tools/02-chromdev.md
-Shell scripts, doubles as CLI 201: https://github.com/jankeLearning/content-md/blob/master/dev-knowledge/04-sh-scripts.md
+There are three ways to open the Developer Tools menu:
+1. From the Browser Menu:
+  - Chrome: Select the `Chrome Menu` > `More Tools` > `Developer Tools`
 
-- evan
\ No newline at end of file
+  - Firefox: Select the Firefox `Menu` > Click on the wrench (`Developer`) icon > `Toggle Tools`
+2. Right click anywhere on a webpage and select `Inspect`
+3. Use the keyboard shortcut `CTRL + Shift + I` (`option + command + I` on Mac)
+
+## Your Assignment
+
+1. Watch [this video](https://www.youtube.com/watch?v=wcFnnxfA70g) by Jose Vidal on how to use the Developer Tools menu.
+2. Read over the [Chrome DevTools Documentation](https://developer.chrome.com/devtools) by Google. Pay particular attention to sections "Accessing the DevTools" through "Debugging Javascript" since this is what you'll use the Developer Tools for 95% of the time.
+3. Learn to inspect HTML page elements by reading through [this article](https://developers.google.com/web/tools/chrome-devtools/inspect-styles/) by Google, then continue on to [Edit the DOM](https://developers.google.com/web/tools/chrome-devtools/inspect-styles/edit-dom)
```

Comment:
let's move assignments 3 and 4 to additional resources because they are HTML/CSS focused and _I think_ that we should have already covered that by the time they get here... so we should reference them below as a refresher, but keep the assignment JS focused.